apps/web/lib/components/kanban-card.tsx

```diff
@@ -154,9 +149,7 @@ export default function Item(props: ItemProps) {
 			alt: member.user.firstName
 		};
 	});
-	const { collaborativeSelect } = useCollaborative(memberInfo.memberUser);
 
-	const menu = <>{!collaborativeSelect && <UserTeamCardMenu memberInfo={memberInfo} edition={taskEdition} />}</>;
```

Comment:
I've removed the menu from # 159 and added the whole component into line no # 180